notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-1165_m1m3_bump_test_error_and_measured_forces.ipynb

```diff
@@ -0,0 +1,1060 @@
+    "    start = datetime.fromisoformat(start_date)\n",
+    "    end = datetime.fromisoformat(end_date)\n",
+    "\n",
+    "    # Initialize an empty DataFrame to store concatenated results\n",
+    "    all_data = pd.DataFrame()\n",
+    "\n",
+    "    current_start = start\n",
+    "    while current_start < end:\n",
+    "        current_end = min(current_start + timedelta(days=chunk_size_days), end)\n",
+    "        try:\n",
+    "            # Query the data for the current chunk\n",
+    "            chunk_data = await client.select_time_series(\n",
+    "                topic_name=\"lsst.sal.MTM1M3.logevent_logMessage\",\n",
+    "                fields=[\"message\"],\n",
+    "                start=Time(current_start.isoformat(), format=\"isot\", scale=\"utc\"),\n",
+    "                end=Time(current_end.isoformat(), format=\"isot\", scale=\"utc\"),\n",
+    "            )\n",
+    "            # Concatenate the chunk data to the main DataFrame\n",
+    "            all_data = pd.concat([all_data, chunk_data], ignore_index=False)\n",
+    "        except Exception as e:\n",
+    "            print(\n",
+    "                f\"Error querying data from {current_start.isoformat()} to {current_end.isoformat()}: {e}\"\n",
+    "            )\n",
+    "            continue  # Optionally, continue to the next chunk\n",
+    "\n",
+    "        # Move to the next chunk\n",
+    "        current_start = current_end\n",
+    "\n",
+    "    return all_data\n",
+    "\n",
+    "\n",
+    "# Example usage:\n",
+    "# begin = \"2023-11-13T01:00\"\n",
+    "# end = \"2023-12-21T01:00\"\n",
+    "# bump_logs = await query_bump_logs_in_chunks(begin, end, client_name='')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f5d39c85-e841-46e3-acad-136e82cc0582",
+   "metadata": {},
+   "source": [
+    "### Making Client and Checkig schema"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8e2d2b94-b94a-4293-996b-d9e8a051e323",
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2024-02-05T18:54:17.663902Z",
+     "iopub.status.busy": "2024-02-05T18:54:17.663622Z",
+     "iopub.status.idle": "2024-02-05T18:54:18.125049Z",
+     "shell.execute_reply": "2024-02-05T18:54:18.124631Z",
+     "shell.execute_reply.started": "2024-02-05T18:54:17.663888Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# Schema may not work depending on the client (e.g. it does not work on `idf`) and how you execute your notebook (e.g. via vscode ssh connection)\n",
+    "client = makeEfdClient()\n",
+    "schema = await client.get_schema(\"lsst.sal.MTM1M3.logevent_logMessage\")\n",
+    "schema"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7ad29d05-8221-4925-98ad-630e50bd267b",
+   "metadata": {},
+   "source": [
+    "### Getting EFD data\n",
+    "\n",
+    "Here we just run helpper function defined above to get the log messages from the EFD."
```

Comment:
Maybe one quick warning like 
"This can take a longer time depending on the begin and end date and time".